Backend/app/rider-platform/rider-app/Main/spec/Storage/ClientPersonInfo.yaml

```
@@ -3,28 +3,28 @@ imports:
   Merchant: Domain.Types.Merchant
   Client: Domain.Types.Client
   Person: Domain.Types.Person
-  Merchant : Domain.Types.Merchant
-  VehicleCategory : BecknV2.OnDemand.Enums
+  Merchant: Domain.Types.Merchant
```

yamllint (1.35.1)

[error] duplication of key "Merchant" in mapping

(key-duplicates)